packages-nextgen/kestrel_core/src/kestrel/frontend/compile.py

```diff
@@ -61,17 +62,61 @@ def _create_comp(field: str, op: str, value):
     return comp
 
 
+def _map_filter_exp(entity_name, filter_exp, property_map):
```

Comment:
Can we add types for the arguments and return, and the `@typechecked` decorator?

Reply:
Addressed in commit https://github.com/opencybersecurityalliance/kestrel-lang/pull/434/commits/182d42e746fc877f7fc111550e699d126b17aa00